builder/main.py

```diff
@@ -65,7 +65,8 @@ def _rpi_sysgpio(path, value):
         _rpi_sysgpio("/sys/class/gpio/gpio%d/value" % pin_num, 0)
         _rpi_sysgpio("/sys/class/gpio/unexport", pin_num)
     else:
-        if not upload_options.get("disable_flushing", False):
+        if not upload_options.get("disable_flushing", False) \
+            and not env.get("UPLOAD_PORT").startswith("net:"):
```

Comment:
Please replace this line with `and not env.get("UPLOAD_PORT", "").startswith("net:")` to avoid exception when UPLOAD_PORT is not available